.buildkite/pipelines/on_merge.yml

```diff
@@ -86,6 +86,14 @@ steps:
     agents:
       queue: 'kibana-default'
 
+  - command: '.buildkite/scripts/steps/functional/on_merge_serverless_ftrs.sh'
```

Comment:
@pheyos I know I'm way late to this PR party, but I'd like to know if the chronological position of this command is significant?  I noticed it's just after `on_merge_unsupported_ftrs.sh` and just before `lint`.  

Really, just curious.